scripts/legislation/africa/rwanda_french.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+function remove_all_text_before_first_chapter_header {
+  sed -n '/CHAPITRE PREMIER - DES DISPOSITIONS GENERALES /,$p'
+}
+
+function amend_errors_in_headers {
+  sed -E 's/^(Section [0-9]+ - [A-Za-z,’ ]+)[^a-z]+Article ([0-9]+)(er)?/\1\n\n(\2)/' | \
```

Comment:
Does your branch include [this](https://github.com/danpdraper/eia/commit/a1d55982c071e5e0675a4e90c145a37ab5ff18b9) commit? If it does not, please pull mainline and rebase your local issue branch. I ask because the forward tick in your first captured group should be a single quotation mark.

Reply:
Yes I do have that commit, I have changed the forward tick to be a single quotation mark.